Sources/StreamChat/ChatClient.swift

```diff
@@ -132,6 +132,10 @@ public class ChatClient {
         environment.callRepositoryBuilder(apiClient)
     }()
 
+    func makeMessagesPaginationStateHandler(parentMessageId: MessageId?) -> MessagesPaginationStateHandling {
```

Comment:
You mean the `parentMessageId`? I would, but I'm actually going to remove this in a follow-up PR. After I fix a bug, we don't actually need this parentMessageId any more, so that is why I didn't explain it here. 👍

Reply:
I meant in general to add a comment for the method but it's internal so I'm alright not having one.